packages/webdriverio/tests/commands/browser/call.test.js

```diff
@@ -20,6 +20,12 @@ describe('call command', () => {
     })
 
     it('should not fail if nothing is applied', () => {
-        browser.call()
+        expect(() => browser.call())
```

Comment:
should not fail, but you expect it to throw

Reply:
if you don't pass a function in it is indeed suppose to fail